modules/model/shared/src/main/scala/observe/model/package.scala

```diff
@@ -38,6 +40,21 @@ extension (i: Instrument)
 //      case Instrument.Gpi   => true
 //      case Instrument.Ghost => false
 
+  def site: Option[Site] = i match
```

Comment:
Does iit need to be an option? instruments always have a site?

Reply:
The `Instrument` enum has things like `Michelle`, `Visitor`, `Phoenix`, etc.

Reply:
@swalker2m Why do we have those ancient instruments, are we planning to port the old ocs data?